### PR TITLE
Add missing build dependencies

### PR DESCRIPTION
**Pull request by dhoppe**
These changes are necessary to build Docker images for Raspberry Pi OS (32-bit).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,20 @@
 FROM ubuntu:20.04
 LABEL maintainer="Jeff Geerling"
 
+ENV DEBIAN_FRONTEND noninteractive
```
Comment (dhoppe) [inline Dockerfile:4]: This environment variable is also used for Debian 9/10. It also solves the problem of #2 or #3.

**Comment by c-goes** [comment]
I would have preferred ARG over ENV for the noninteractive variable. So this environment only applies for the build.

**Comment by dhoppe** [comment]
Makes sense. Looking at the issues, the Dockerfile‘s could basically use some attention.

**Comment by geerlingguy** [comment]
@c-goes - Ah, true. Happy to field another PR :)

Also @dhoppe I am now pushing an arm64 image in addition to the amd64!

**Pull request merged**